src/common.tests/Utilities/StringManipulationTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.BridgeToKubernetes.Common.Utilities;
+using Xunit;
+
+namespace Microsoft.BridgeToKubernetes.Common.Tests.Utilities
+{
+    public class StringManipulationTests
+    {
+        [Theory]
+        [InlineData("BEGIN PRIVATE KEY\nuirethweuifhewiufhweiuofhweo\njweiorjweoifnewouhfousdgherufhsdiuhEND PRIVATE KEY\n", "KEY_WAS_REMOVED\n")]

```

Comment:
can you add one more InlineData for ```BEGIN PRIVATE KEYb2kkkkkkkkkkkkkkEND PRIVATE KEY``` without ```\n``` break lines ? just to make sure different use cases work.